### ISSUE TITLE
[*] Lumber Generate - Specifying option --email does not work if already logged in

### ISSUE THREAD

**Issue by VincentMolinie** [ForestAdmin/lumber#228]
### What happen
Using the option `--email` with an email different than the one use in lumber login (I think lumber login is the old version) ignore the email, does not ask password and create the project on the other accoun.

### What I expect
I should be asked to enter my password for that email and should create the project for that account

**Comment by hellokvn** [comment]
Same here. I created a project, plugged the MySQL database and did the same with another ForestAdmin account. When it came to this command:

`lumber install lumber-forestadmin --email avyjov@clrmail.com --projectName loremipsum`

the E-Mail address of my new ForestAdmin account was ignored complete, instead I created new projects with this command on my previous ForestAdmin account.

Windows 10 64 bit
Node 10.14.1
NPM 6.4.1
